atlasdb-perf/README.md

```diff
@@ -0,0 +1,12 @@
+# AtlasDB Performance CLI
+
+The AtlasDB Performance CLI is an extendable framework for creating and running performance benchmarks against AtlasDB services.
+
+## Running Tips
+
+`//TODO`
```

Comment:
haha, so quick =)

I'll fix this.  I made a pull request because this commit also magically breaks Intellij and I wanted it to build against circle.